### PR TITLE
Implement SwingBrowserView

### LINKED ISSUE #8: Test Swing BrowserView in Compose Desktop app
Please integrate Swing `BrowserView` into a Compose Desktop app with address bar and check the following functionality:

1. Use the off-screen rendering mode.
2. Show/hide and bounds.
3. Z-order and overlapping.
4. Focus traversal (Tab, Shift+Tab, mouse click, `requestFocus()`).
5. Rendering performance. Is it the same as in Swing or different?
6. Drag and drop.
7. HiDPI.
8. JS alert, confirm, prompt, color picker.
9. File Open/Save dialogs.
10. Print Preview.
11. Drop downs on a web page.

### PR DESCRIPTION
**Pull request by Danil-Didkovskiy**
Fixes #8 

In this changeset, we implement the `SwingBrowserView` component that embeds our `com.teamdev.jxbrowser.view.swing.BrowserView` in Compose `SwingPanel`. 

We tested how our Swing `BrowserView` component behaves in the Compose environment. It supports all the necessary functionality, but we found one obvious problem - if it is embedded in `SwingPanel`, it is always heavyweight (regardless of the rendering mode). The [Swing integration](https://github.com/JetBrains/compose-jb/tree/master/tutorials/Swing_Integration) in Compose only works in that way. Therefore, we decided to keep both implementations for now.

### CODE REVIEW

**Comment by Danil-Didkovskiy** [comment]
@vladimir-ikryanov PTAL

**Comment by vladimir-ikryanov** [comment]
@Danil-Didkovskiy Please fill in the following required PR fields:

- Reviewers
- Assignees (it's you)

**Comment by vladimir-ikryanov** [comment]
@Danil-Didkovskiy I think we shouldn't mix both implementations in a single file. There's should be some hierarchy like:

```
             BrowserView
             |       |
ComposeBrowserView SwingBrowserView
```

**Review by vladimir-ikryanov — APPROVED** [review]

File: src/main/kotlin/com/teamdev/jxbrowser/compose/BrowserView.kt [hunk 1]
```diff
@@ -15,108 +15,23 @@
...
-private const val STOP_PROPAGATION = true
+import com.teamdev.jxbrowser.engine.RenderingMode.HARDWARE_ACCELERATED
+import com.teamdev.jxbrowser.engine.RenderingMode.OFF_SCREEN
 
 /**
- * Adds a widget that displays the content of the [Browser].
+ * Adds a component that displays the content of the [Browser].
+ *
+ * The rendering mode determines the type of component. If the rendering mode is [OFF_SCREEN],
+ * it adds the [ComposeBrowserView] component built with Compose only. On [HARDWARE_ACCELERATED],
```
Comment (vladimir-ikryanov) [inline src/main/kotlin/com/teamdev/jxbrowser/compose/BrowserView.kt:28]: Please don't share the implementation details in Javadoc.

**Review by vladimir-ikryanov — CHANGES_REQUESTED** [review]

**Review by vladimir-ikryanov — APPROVED** [review]

**Pull request merged**